### PR TITLE
JAXWS install and setup guide updated.

### PR DESCRIPTION
**Pull request by Tomas-Kraus**
Tomcat setup documentation updated.

### CODE REVIEW

**Comment by Tomas-Kraus** [comment]
Here is [XML-WS-TCK-Users-Guide.pdf](https://github.com/eclipse-ee4j/jakartaee-tck/files/5102871/XML-WS-TCK-Users-Guide.pdf) matching this PR.

**Comment by Tomas-Kraus** [comment]
Updated [XML-WS-TCK-Users-Guide.pdf](https://github.com/eclipse-ee4j/jakartaee-tck/files/5103278/XML-WS-TCK-Users-Guide.pdf)

**Comment by Tomas-Kraus** [comment]
Updated [XML-WS-TCK-Users-Guide.pdf](https://github.com/eclipse-ee4j/jakartaee-tck/files/5103532/XML-WS-TCK-Users-Guide.pdf)

**Comment by Tomas-Kraus** [comment]
Updated [XML-WS-TCK-Users-Guide.pdf](https://github.com/eclipse-ee4j/jakartaee-tck/files/5117724/XML-WS-TCK-Users-Guide.pdf)

File: user_guides/jaxws/src/main/jbake/content/config.inc [hunk 1]
```diff
@@ -43,8 +43,8 @@ slashes as a path separator instead.
   {TechnologyVersion} software is installed
   c.  `PATH` to include the following directories: `JAVA_HOME/bin`,
   +{TechnologyHomeEnv}/bin+, and `ANT_HOME/bin`
-2.  Edit your `<TS_HOME>/bin/ts.jte` file and set the following
-environment variables:
+2.  If you use Jakarta EE 9 Compatible Platform, edit your `<TS_HOME>/bin/ts.jte`
```
Comment (edbratt) [inline user_guides/jaxws/src/main/jbake/content/config.inc:46]: You could replace the Jakarta EE version ID with {JakartaEEVersion}

File: user_guides/jaxws/src/main/jbake/content/config.inc [hunk 2]
```diff
@@ -181,55 +250,33 @@ environment variables:
   d.  Set the `jaxws.classes` property to point to the Vendor
   Implementation classes/JAR files. +
   As an example, the `ts.jte` file contains the property
-  `jaxws.classes.ri`, which contains the classes/jar files that the Jakarta EE 9 Compatible Platform uses. The `jaxws.classes.ri` settings for the Jakarta EE {JakartaEEVersion} Compatible Platform web
-  container are as follows:
+  `jaxws.classes.ri`, which contains the classes/jar files that the Jakarta EE 9
```
Comment (edbratt) [inline user_guides/jaxws/src/main/jbake/content/config.inc:253]: Consider changing the version to property (as above)

File: user_guides/jaxws/src/main/jbake/content/config.inc [hunk 3]
```diff
@@ -181,55 +250,33 @@ environment variables:
...
-${jaxws.lib.ri}/saaj-impl.jar:${jaxws.lib.ri}/sdo-eclipselink-plugin.jar:\
-${jaxws.lib.ri}/stax2-api.jar:${jaxws.lib.ri}/stax-ex.jar:\
-${jaxws.lib.ri}/streambuffer.jar:${jaxws.lib.ri}/woodstox-core-asl.jar:
-----
---
-+
-  e.  Verify that the `tools.jar` property is set to the location of the
-  `tools.jar` file that is distributed with Java SE.
-  f.  Set the `impl.vi`, `impl.vi.deploy.dir`, `impl.vi.host`, and
+with the Standalone XML-WS 3.0 Compatible Implementation should contain
```
Comment (edbratt) [inline user_guides/jaxws/src/main/jbake/content/config.inc:277]: consider replacing all references of JAX-WS with {TechnologyShortName}

File: user_guides/jaxws/src/main/jbake/content/config.inc [hunk 4]
```diff
@@ -273,15 +320,15 @@ com.sun.ts.lib.implementation.sun.common.SunRIURL
...
   The default setting for both is `j2ee`.
-  j. Set the `authuser` and `authpassword` properties to the user name
+  i. Set the `authuser` and `authpassword` properties to the user name
   and password used for the basic authentication tests. +
   The default setting for both is `javajoe`.
-  k. Set the `http.server.supports.endpoint.publish` property based on
+  j. Set the `http.server.supports.endpoint.publish` property based on
   whether Endpoint Publish APIs are supported on the container.
-  l. If using Java SE 8 or above, verify that the property
+  k. If using Java SE 8 or above, verify that the property
```
Comment (edbratt) [inline user_guides/jaxws/src/main/jbake/content/config.inc:331]: consider changing Java SE version ID to {SEversion}

File: user_guides/jaxws/src/main/jbake/content/config.inc (hunk 1 above)
Comment (Tomas-Kraus) [reply user_guides/jaxws/src/main/jbake/content/config.inc:46]: Yes, done.

File: user_guides/jaxws/src/main/jbake/content/config.inc (hunk 2 above)
Comment (Tomas-Kraus) [reply user_guides/jaxws/src/main/jbake/content/config.inc:253]: OK

File: user_guides/jaxws/src/main/jbake/content/config.inc (hunk 3 above)
Comment (Tomas-Kraus) [reply user_guides/jaxws/src/main/jbake/content/config.inc:277]: TechnologyShortName is "XML Web Services".  But yes, I'll try it with combination of TechnologyVersion too.

Comment (Tomas-Kraus) [reply user_guides/jaxws/src/main/jbake/content/config.inc:277]: Done

File: user_guides/jaxws/src/main/jbake/content/config.inc (hunk 4 above)
Comment (Tomas-Kraus) [reply user_guides/jaxws/src/main/jbake/content/config.inc:331]: I have to remove this paragraph. Endorsed dir makes no sense with all packages renamed to jakarta.*

Comment (Tomas-Kraus) [reply user_guides/jaxws/src/main/jbake/content/config.inc:331]: But also did search/replace for Java SE 8 in the rest of this file.

**Comment by Tomas-Kraus** [comment]
Updated [XML-WS-TCK-Users-Guide.pdf](https://github.com/eclipse-ee4j/jakartaee-tck/files/5123660/XML-WS-TCK-Users-Guide.pdf)

I extended parametrization of config.inc a bit but I'd like not to touch other files now.
Also please make sure that I did not put any nonsense into the document. I'm not native English speaker so there is always a chance for this to happen. :)

**Comment by Tomas-Kraus** [comment]
@senivam @lukasj Please have a look at those changes too.
I verified both GF and Tomcat guides by setting up tests and running them following the guide. Both runs passed with GF6 / Tomcat with latest JAX-WS release.

**Comment by Tomas-Kraus** [comment]
[XML-WS-TCK-Users-Guide.pdf](https://github.com/eclipse-ee4j/jakartaee-tck/files/5123730/XML-WS-TCK-Users-Guide.pdf)

**Comment by Tomas-Kraus** [comment]
[XML-WS-TCK-Users-Guide.pdf](https://github.com/eclipse-ee4j/jakartaee-tck/files/5123752/XML-WS-TCK-Users-Guide.pdf)

**Review by senivam — APPROVED** [review]
Looks good to me

**Pull request merged**